packages/zowe-explorer/CHANGELOG.md

```diff
@@ -2,18 +2,23 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
-## `1.18.1`
-
-- Improved integration tests for greater efficiency [#1430](https://github.com/zowe/vscode-extension-for-zowe/pull/1430). Thanks @katelynienaber
-- Added a reminder to the PR template to create a PR from `master` to `next` after PR is merged into `master` branch [#1446](https://github.com/zowe/vscode-extension-for-zowe/pull/1446). Thanks @JillieBeanSim
-- Added @zowe/cli dependency to API [#1445](https://github.com/zowe/vscode-extension-for-zowe/pull/1445). Thanks @phaumer
-- Fix for the issue that caused mismatching of GitHub VSIX Zowe Explorer releases [#1442](https://github.com/zowe/vscode-extension-for-zowe/pull/1442). Thanks @katelynienaber
+## `1.19.0`
+
+- Added the function that checks whether a base profile exists before running the function that combines base and service profiles [#1500](https://github.com/zowe/vscode-extension-for-zowe/pull/1500). Thanks @lauren-li
+- Added Imperative logger access for extenders [#1433](https://github.com/zowe/vscode-extension-for-zowe/pull/1433). Thanks @katelynienaber
+- Added documentation for Imperative logger for extenders [#1467](https://github.com/zowe/vscode-extension-for-zowe/pull/1467). Thanks @katelynienaber
+- Added the profile validation and credentials check functionalities to the Zowe Explorer FTP extension [#1284](https://github.com/zowe/vscode-extension-for-zowe/issues/1284). Thanks @tiantn
```

Comment:
@IgorCATech Do we want to move this item to the FTP extension's Readme/Changelog pages?:
- [FTP Extension Readme](https://github.com/zowe/vscode-extension-for-zowe/tree/master/packages/zowe-explorer-ftp-extension#readme)
- [FTP Extension Changelog](https://github.com/zowe/vscode-extension-for-zowe/blob/master/packages/zowe-explorer-ftp-extension/CHANGELOG.md)

Reply:
I think we can but it's probably dependent on our 'marketing strategy'. What I mean is do we want to advertize these changes in the ZE readme and changelog or should we place these updates to the FTP readme? 
Also, in order to avoid confusion, should we mention these items as part of the FTP changes? The release versions do not match, though. 
@jellypuno any thought on this one?

Reply:
@IgorCATech Traditionally, we have put the FTP ZE extension updates separately from the core Zowe Explorer updates, which is also helpful for when users search for it in the VS Code Marketplace.

Reply:
@lauren-li I think it makes sense. Going forward, though, I wouldn't mention zFTP-related PRs in ZE releases. Does that sound reasonable?

Reply:
I agree that FTP updates shouldn't be in both, Zowe Explorer and the FTP ext each have their own marketplace page that should list it's specific updates and not cross reference.

Reply:
@IgorCATech I actually don't recall us mentioning zFTP-related PRs in core ZE releases previously. Have we done that before? I thought we have always listed them separately in their respective repos.

Reply:
@IgorCATech I agree, it's confusing when other package numbers fall behind due to not having an update with every Zowe Explorer release. I wouldn't mind adopting eslint-plugin's changelog strategy.